homeworks/sargis_vardanyan/NewMyChat/client/ClientWorkerThread.java

```diff
@@ -0,0 +1,30 @@
+package client;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.Socket;
+
+class ClientWorkerThread extends Thread {
+    private Socket socket;
+
+    public ClientWorkerThread(Socket socket) {
+        this.socket = socket;
+    }
+
+    @Override
+    public void run() {
+        while (true) {
+            try {
+                BufferedReader reader = new BufferedReader(new InputStreamReader(socket.getInputStream()));
+                String line = reader.readLine();
+                while (line != null) {
+                    System.out.println(line);
+                    line = null;
```

Comment:
if client send the following message what would happen:
printwriter.println("hello \n, how are you doing")
what will happen with your code?